src/riak_kv_wm_utils.erl

```diff
@@ -355,6 +355,8 @@ jsonify_bucket_prop({search_extractor, {M, F, Arg}}) ->
                               {?JSON_ARG, Arg}]}};
 jsonify_bucket_prop({name, {_T, B}}) ->
     {<<"name">>, B};
+jsonify_bucket_prop({ddl, _DDL}) ->
+    {<<"ddl">>, <<"riak_ql_to_string:sql_to_string might be useful here.">>};
```

Comment:
It's a stub, effectively. And, it's not really important because the DDL structure can is available via a DESCRIBE query.